accounts/src/main/resources/application.yml

```diff
@@ -12,4 +12,11 @@ spring:
     config:
       name: shop
   config:
-    import: configserver:http://localhost:8888
\ No newline at end of file
+    import: configserver:http://localhost:8888
+
+# can delete
+email:
+  host: ${RSS_EMAIL_HOST:localhost}
+  port: ${RSS_EMAIL_PORT:25}
+  username: ${RSS_EMAIL_USERNAME:user}
+  Password: ${RSS_EMAIL_PASSWORD:password}
```

Comment:
what's this?

Reply:
Will remove now. May have added those while trying to find solutions to some issues that weren't issues in the first place.